omod/src/main/java/org/openmrs/module/owa/filter/OwaFilter.java

```diff
@@ -51,8 +62,15 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
 		} else {
 			requestURL = request.getServletPath();
 		}
+		String loginUrl = "";
+		try {
+			Context.addProxyPrivilege("Get Global Properties");
+			loginUrl = Context.getAdministrationService().getGlobalProperty("login.url", "login.htm");

```

Comment:
Are you intentionally avoiding to catch this exception?

Reply:
Since we are adding the proxy privilege, I don't think it's necessary to catch any exceptions here.